templates/bootstrap.sh

```diff
@@ -436,7 +436,11 @@ else
     # Build the Docker images using Compose, namespaced to the job
     echo "~~~ Building Docker images"
 
-    buildkite-run "$COMPOSE_COMMAND build"
+    if [[ "${BUILDKITE_DOCKER_COMPOSE_BUILD_ALL:-false}" == "true" ]]; then
```

Comment:
Thanks for the ProTip:tm:, @caarlos0! The `:-false` and `==` seem to follow the coding style present in the file, so I'll wait for @toolmantim to have a final say about this.